test/utils.c

```diff
@@ -166,6 +169,18 @@ TEST test_string_to_path(void)
         free(exp);
         free(ptr);
 
+        ASSERT_STR_EQ((exp = g_strconcat(home, "/.path/with/HOME environment variable", NULL)),
+                      (ptr = string_to_path(g_strdup("$HOME/.path/with/HOME environment variable"))));
+        free(exp);
+        free(ptr);
+
+// At least musl doesn't properly support wordexp.
```

Comment:
That seems like a problem, since we do support musl

Reply:
The problem here is, that `wordexp` on musl just does not expand the value if the variable is unset. In case the variable is know, it works as intended (the other tests in this file pass on musl as intended).

Reply:
Ah okay. Could you clarify the comment then?